src/PairEvaluatorPerturbedLennardJones.h

```diff
@@ -0,0 +1,176 @@
+// Copyright (c) 2018-2020, Michael P. Howard
+// Copyright (c) 2021-2024, Auburn University
+// Part of azplugins, released under the BSD 3-Clause License.
+
+#ifndef AZPLUGINS_PAIR_EVALUATOR_PERTURBED_LENNARD_JONES_H_
+#define AZPLUGINS_PAIR_EVALUATOR_PERTURBED_LENNARD_JONES_H_
+
+#include "PairEvaluator.h"
+
+#ifdef __HIPCC__
+#define DEVICE __device__
+#define HOSTDEVICE __host__ __device__
+#else
+#define DEVICE
+#define HOSTDEVICE
+#endif
+
+namespace hoomd
+    {
+namespace azplugins
+    {
+namespace detail
+    {
+//! Define the parameter type used by this pair potential evaluator
+struct PairParametersPerturbedLennardJones : public PairParameters
```

Comment:
This struct implements the conversion between Python and C++. You need to have a default constructor (no arguments) and one that takes a Python dictionary & bool, as below.